core/src/message/context.rs

```diff
@@ -222,29 +229,16 @@ impl MessageContext {
     ) -> Option<Self> {
         let (kind, message, store) = dispatch.into_parts();
 
-        let outgoing_bytes_counter = match &store {
-            Some(store) => {
-                let mut counter = 0u32;
-                for payload in store.outgoing.values().filter_map(|x| x.as_ref()) {
-                    counter = counter.checked_add(payload.len_u32())?;
-                }
-                counter
-            }
-            None => 0,
-        };
-
-        if outgoing_bytes_counter > settings.outgoing_bytes_limit {
-            // Outgoing messages bytes limit exceeded.
-            return None;
-        }
-
         Some(Self {
             kind,
             outcome: ContextOutcome::new(program_id, message.source(), message.id()),
             current: message,
             store: store.unwrap_or_default(),
+            outgoing_payloads: OutgoingPayloads::default(),
             settings,
-            outgoing_bytes_counter,
+            // message context *always* starts with zero bytes outgoing. Before it could be non-zero and eventually
```

Comment:
counter is better to move to OutgoingPayloads itself since in facts it's the size of OutgoingPayloads

Reply:
so it will be named just bytes_counter